src/app/course-page/course-search/course-search.component.scss

```diff
@@ -0,0 +1,17 @@
+.course-search {
+  &-container {
+    margin: 20px 0;
+  }
+}
+
+input {
```

Comment:
please use a descriptive class here, so when you read the styles you definitely know what it styles
`.search-input`

Reply:
done